Changelog.md

```diff
@@ -2,6 +2,7 @@
 ### Added
 - **Breaking Change** `.parameterEncoding(Swift.Error)` case to `MoyaError`. [#1248](https://github.com/Moya/Moya/pull/1248) by [@SD10](https://github.com/SD10).
 - **Breaking Change** `Decodable` object mapping methods to `Moya.Response`. [#1335](https://github.com/Moya/Moya/pull/1335) by [@devxoul](https://github.com/devxoul).
+- `.requestJSONEncodable` case to  `Task`. [#1349](https://github.com/Moya/Moya/pull/1349) by [LeLuckyVint](https://github.com/LeLuckyVint), [@afonsograca](https://github.com/afonsograca) and [@sunshinejr](https://github.com/sunshinejr).
```

Comment:
You missed the `@` when tagging @LeLuckyVint :wink:

Reply:
:D you always spot my little errors that nobody notices, thanks!